src/errors/BundlrError.ts

```diff
@@ -0,0 +1,57 @@
+import { MetaplexError, MetaplexErrorInputWithoutSource } from './MetaplexError';
+
+export class BundlrError extends MetaplexError {
+  constructor(input: MetaplexErrorInputWithoutSource) {
+    super({
+      ...input,
+      key: `plugin.bundlr.${input.key}`,
+      title: `Bundlr > ${input.title}`,
+      source: 'plugin',
+      sourceDetails: 'Bundlr',
+    });
+  }
+}
+
+export class FailedToInitializeBundlrError extends BundlrError {
+  constructor(cause: Error) {
+    super({
+      cause,
+      key: 'failed_to_initialize_bundlr',
```

Comment:
What are the keys useful for now?
Given we can detect the specific error via `instanceof` we might not need this anymore?

Reply:
I've left it for localisation. It doesn't cost much to do it now vs it will be a pain to update everything later. What do you think?

Reply:
Seems superfluous to me, but also doesn't hurt anything I suppose.